src/cpp/hu.bme.mit.inf.modes3.ESP32-sensor/railsensor_3/ESP32-sensor/lib/SensorMonitor/sensor_monitor.cpp

```diff
@@ -0,0 +1,230 @@
+
+#include "sensor_monitor.h"
+
+
+float convertRawGyro(int gRaw) {
+  // since we are using 250 degrees/seconds range
+  // -250 maps to a raw value of -32768
+  // +250 maps to a raw value of 32767
+
+  float g = (gRaw * 250.0) / 32768.0;
+
+  return g;
```

Comment:
You could simple return the result of the calculation from above. Don't need the local variable.